minerl/herobraine/hero/mc.py

```diff
@@ -672,3 +672,71 @@ def strip_item_prefix(minecraft_name):
         return minecraft_name[len('minecraft:'):]
 
     return minecraft_name
+
+
+# Transcribed from: https://minecraft.gamepedia.com/index.php?title=Biome&oldid=1057817,
+#   which has the biomes as they were during the minecraft version at the time: 1.11.2
+# _m indicates that the terrain is more mountainous
+# _f indicates that the terrain has trees where the base biome does not.
+BIOMES_MAP = {
+    "ice_plains": 12,
+    "ice_plains_spikes": 140,
```

Comment:
Check that biome map above 32 is not off by 2. (Brandon says that that above 32 is off by 2)

Reply:
Seems legit!

![image](https://user-images.githubusercontent.com/1750835/115099407-78bb2a80-9eea-11eb-86d3-baab4bb25e4b.png)

Reply:
Sweet! Looks spot on to me